target/linux/mediatek/patches-5.4/0999-hnat.patch

```diff
@@ -147,25 +147,6 @@
  
  #include "mtk_eth_soc.h"
  
-@@ -76,6 +78,18 @@ u32 mtk_m32(struct mtk_eth *eth, u32 mas
- 	return reg;
- }
- 
-+void mtk_m32(struct mtk_eth *eth, u32 clear, u32 set, unsigned reg)
-+{
-+	u32 val;
-+
-+	spin_lock(&eth->page_lock);
-+	val = __raw_readl(eth->base + reg);
-+	val &= ~clear;
-+	val |= set;
-+	__raw_writel(val, eth->base + reg);
-+	spin_unlock(&eth->page_lock);
```

Comment:
The function in the upstream kernel looks different:
´´´
u32 mtk_m32(struct mtk_eth *eth, u32 mask, u32 set, unsigned reg)
{
	u32 val;

	val = mtk_r32(eth, reg);
	val &= ~mask;
	val |= set;
	mtk_w32(eth, val, reg);
	return reg;
}
´´´
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=1ff0732cf8055131050fa0c26c49429d6e725505

Reply:
Oh, I missed spinlock. I'll patch it. (except that, they are identical)

Reply:
OK, fixed.